lib/auth/github_authenticator.rb

```diff
@@ -113,4 +117,15 @@ def register_middleware(omniauth)
            },
            scope: "user:email"
   end
+
+  protected
```

Comment:
It is unlikely, at least for the moment, that someone would create a subclass from this class. Can we make this a private method instead? I would just add the test cases into one of the existing test case where the authentication is successful.